docs/src/user-docs/quick-start/clp-json.md

```
@@ -11,22 +11,129 @@ text logs, refer to [this section below](#compressing-unstructured-text-logs).
-:::{tip}
+```{tip}
 To validate configuration and prepare directories without launching services, add the
 `--setup-only` flag (e.g., `sbin/start-clp.sh --setup-only`).
-:::
+```
 
-:::{note}
+```{note}
 If CLP fails to start (e.g., due to a port conflict), try adjusting the settings in
 `etc/clp-config.yaml` and then run the start command again.
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)